commercial examples

woodbe opened the pull request:
I have added additional columns for each table and added some specific examples I could find easily (or know). I added text from @gfiumara about not being "you must use these".

This is to close #10. It is also related to #2

Commit:
commercial examples

I have added additional columns for each table and added some specific examples I could find easily (or know). I added text from @gfiumara about not being "you must use these".

This is to close #10. It is also related to #2

gfiumara commented on FINGERPRINT_Toolbox_Inventory.adoc:
```diff
@@ -34,6 +37,7 @@ a|Commercial fingerprint scanner
 * 1000 dpi minimum image resolution
 * Rolled images preferred
 * Touch/slap images acceptable
+a|
```
Let's get examples of easily-purchasable 1000 PPI scanners from @gregott in here.

woodbe replied:
I completely agree, but didn't know if we should have them added in this PR or if they could be added later once this is approved (since this is more about the formatting with my two examples added) than it is about the actual set of examples which @gregott probably needs to provide.

Commit:
editing copper thickness

I have added updated text in the inventory to specifically state the board should have either the 1oz or 2oz sizes (using all three thickness measures).

I didn't change the text in the attack since I expect that making it clear in the inventory should cover the choices.

I edited the text to bring the one line into proper alignment in the attacks (4 of 5 needed it)

This should close #12

Commit:
Merge pull request #27 from biometricITC/copper-thickness

editing copper thickness
